### PR TITLE
[CMSP-519] Tested up to WordPress 6.3

### PR DESCRIPTION
**Pull request by pwtyler**
- Adds CI tests against nightly WordPress as well
- fixup broken travis badge in wp.org README (I think these only show on github anyway)

- [x] TODO: Update behat fixture to 6.3

### CODE REVIEW

**Comment by pwtyler** [comment]
Attempting to address upstream issues with https://github.com/pantheon-systems/pantheon-wordpress-upstream-tests/pull/63

File: bin/validate-fixture-version.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    if ! terminus whoami > /dev/null; then
+        if [ -z "${TERMINUS_TOKEN}" ]; then
+            echo "TERMINUS_TOKEN environment variable must be set or terminus already logged in."
+            exit 1
+        fi
+        terminus auth:login --machine-token="${TERMINUS_TOKEN}"
+    fi
+
+
```
Comment (jazzsequence) [inline bin/validate-fixture-version.sh:22]: Nit: Same line break as https://github.com/pantheon-systems/wp-saml-auth/pull/354#discussion_r1291674421

Comment (pwtyler) [inline bin/validate-fixture-version.sh:22]:
```suggestion

```

Comment (pwtyler) [reply bin/validate-fixture-version.sh]: next time just suggest and commit things like this 😆

**Review by rwagner00 — APPROVED** [review]
LGTM

**Pull request merged**